preview_generator/preview/builder/image__wand.py

```diff
@@ -37,10 +37,10 @@
 #         return output
 
 
-def convert_pdf_to_jpeg(pdf: typing.Union[str, typing.IO[bytes]], preview_size: ImgDims) -> BytesIO:
+def convert_pdf_to_jpeg(pdf: typing.IO[bytes], preview_size: ImgDims) -> BytesIO:
 
-    pdf = pdf.read()
-    images = convert_from_bytes(pdf)
+    pdf_content = pdf.read()
```

Comment:
I just avoid the `Union` by using two distinct variables, which is good because the union accepted an `str` parameter, on which we call `read()`.